src/jest/config.json

```diff
@@ -13,7 +13,8 @@
     "!ui_framework/components/**/*/index.js"
   ],
   "moduleNameMapper": {
-    "^ui_framework/components": "<rootDir>/ui_framework/components"
+    "^ui_framework/components": "<rootDir>/ui_framework/components",
+    "\\.(css|less|scss)$": "<rootDir>/src/jest/mockCssFiles.js"
```

Comment:
The [Jest docs](https://facebook.github.io/jest/docs/webpack.html#handling-static-assets) seem to suggests to make a separate mocks for style files than other static assets:
```json
{
    "moduleNameMapper": {
      "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
      "\\.(css|less)$": "<rootDir>/__mocks__/styleMock.js"
    }
}
```

This might be a good idea since with styles we don't care about the returned value:
```js
import 'mystyles.css'
```

whereas with images (and other static assets) we do:
```js
import cat from 'cat.gif'
```

I can add the mocks for styles and assets in general, or just add the mock for the styles. Up to you.